src/content/docs/synthetics/synthetic-monitoring/getting-started/intro-synthetic-monitoring.mdx

```diff
@@ -0,0 +1,320 @@
+---
+title: Introduction to synthetic monitoring
+tags:
+  - Synthetics
+  - Synthetic monitoring
+  - Getting started
+translate:
+  - jp
```

Comment:
I think we can get rid of the translate front matter, but if we keep this here we should probably add Korean just to make sure no one is confused in the future if we ever do a sweep to take them all out.